src/6model/containers.c

```diff
@@ -720,35 +720,37 @@ static void native_ref_configure_container_spec(MVMThreadContext *tc, MVMSTable
 }
 
 void *MVM_container_devirtualize_fetch_for_jit(MVMThreadContext *tc, MVMSTable *st, MVMuint16 type) {
-    if (type != MVM_reg_int64)
-        return NULL;
     if (st->container_spec == &native_ref_spec) {
         MVMNativeRefREPRData *repr_data = (MVMNativeRefREPRData *)st->REPR_data;
         switch (repr_data->ref_kind) {
             case MVM_NATIVEREF_LEX:
                 switch (type) {
                     case MVM_reg_int64: return MVM_nativeref_read_lex_i;
+                    case MVM_reg_uint64: return MVM_nativeref_read_lex_i;
```

Comment:
MVM_nativeref_read_lex_u?

Reply:
Based on https://github.com/MoarVM/MoarVM/blob/abb918cdbfcbead689dbcb1dad8833223c42c610/src/6model/containers.c#L476 I assumed I wouldn't need to add a MVM_nativeref_read_lex_u.